acasclient/experiment.py

```diff
@@ -144,3 +144,13 @@ def original_server(self) -> str:
             value_type="stringValue",
             value_kind=ORIGINAL_SERVER_KEY)
         return data.get('stringValue') if data else None
+    
+    @property
+    def scientist(self) -> str:
+        data = get_entity_value_by_state_type_kind_value_type_kind(
+            entity=self,
+            state_type="metadata",
+            state_kind="experiment metadata",
+            value_type="codeValue",
+            value_kind="scientist")
+        return data.get('codeValue') if data else None
```

Comment:
This is for the tests. I decided to use the Experiment class which we haven't used in tests at all yet but this class simplifies the checking of scientist on the experiment dict so I decided to use it.